modules/hal_nordic/nrfx/nrfx_config.h

```diff
@@ -901,7 +900,7 @@
 #endif
 
 #define NRFX_CONFIG_BIT_DT(node_id, prop, idx) BIT(DT_PROP_BY_IDX(node_id, prop, idx))
-#define NRFX_CONFIG_MASK_DT(node_id, prop) \
+#define NRFX_CONFIG_MASK_DT(node_id, prop)                                                         \
 	(COND_CODE_1(DT_NODE_HAS_PROP(node_id, prop), \
 		(DT_FOREACH_PROP_ELEM_SEP(node_id, prop, NRFX_CONFIG_BIT_DT, (|))), \
 		(0)))
```

Comment:
Do changes like this (there are more similar in this commit) actually improve anything?

Reply:
I was addressing the issues that the compliance clang-format check found. Not a big deal but it's now better aligned with our `.clang-format`

Reply:
I'm not sure it's worth it. IMO at least the above part of code looks more messy than before the change. Perhaps that `.clang-format` should be improved.